gh-80448: argparse: Fix IndexError on store_true action (#15656)
python/cpython: Misc/NEWS.d/next/Library/2019-09-03-15-45-19.bpo-36267.z42Ex7.rst

> Fix IndexError in :class:`argparse.ArgumentParser` when a `store_true` action is given an explicit argument.

Reviewer: Fix RST markup
```suggestion
Fix IndexError in :class:`argparse.ArgumentParser` when a ``store_true`` action is given an explicit argument.
```

> Fix IndexError in argparse when instance of  :class:`ArgumentParser` parse args like '-a=' what have a 'store_true' action.

Reviewer 2: ```suggestion
Fix IndexError in :class:`argparse.ArgumentParser` when a `store_true` action is given an explicit argument.
```

Author: Thanks for your comment. your comment looks more simple and exact.

PS: I have forget this PR's detail in fact~.
